### ISSUE TITLE
--watch seems not to work on OSX on the 0.8.7/0.8.8 version

### ISSUE THREAD

**Issue by sryzycki** [typicode/json-server#241]
Does anyone else has the same issue?

**Comment by thefatoneinthecorner** [comment]
The --watch on version 0.8.10 doesn't seem to work either on OS X (El Capitan 10.11.4).

**Comment by liyatang** [comment]
the same to me . version 0.8.10

**Comment by jon301** [comment]
same here on v0.8.14
OSX 10.11.3

**Comment by liyatang** [comment]
use nodemon。 Look at this  https://github.com/typicode/json-server/issues/286

**Comment by PLQin** [comment]
yes, please use **nodemon**  https://github.com/typicode/json-server/issues/434#issuecomment-689536873
